extensions/positron-python/python_files/posit/positron/data_explorer.py

```diff
@@ -1372,23 +1374,28 @@ def suggest_code_syntax(self, request: SuggestCodeSyntaxRequest):  # noqa: ARG00
         """Returns the supported code types for exporting data."""
         return CodeSyntaxName(code_syntax_name="pandas").dict()
 
-    def convert_to_code(self, request: ConvertToCodeRequest):  # noqa: ARG002
+    def convert_to_code(self, request: ConvertToCodeParams):
         """Translates the current data view, including filters and sorts, into a code snippet."""
-        return ConvertedCode(
-            converted_code=["import pandas as pd", "# TODO: Implement export to code"]
-        ).dict()
+        # hack hack hack, the front end actually does not store sort keys
```

Comment:
should the front end be updating its backend state to track sorts? right now it does not.

Reply:
Poked into this a bit; we update the backend state when [row filters are added](https://github.com/posit-dev/positron/blob/9f9dfd594d04403daf0f0401e14df7153876f141/src/vs/workbench/services/positronDataExplorer/browser/tableDataDataGridInstance.tsx#L685), but [not when sorts are added](https://github.com/posit-dev/positron/blob/9f9dfd594d04403daf0f0401e14df7153876f141/src/vs/workbench/services/positronDataExplorer/browser/tableDataDataGridInstance.tsx#L227). I'll open up a small PR with that update!